src/App.css

```diff
@@ -11,3 +15,18 @@
     padding-top: 70px;
   }
 }
+.pageTitle {
+  text-transform: flex;
```

Comment:
I think `text-transform` doesn't accept `flex` as a value. 
https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Reply:
I have changed to uppercase.